lib/ansible/cli/arguments/option_helpers.py

```diff
@@ -31,6 +31,16 @@ def add_arguments(self, actions):
         super(SortingHelpFormatter, self).add_arguments(actions)
 
 
+class ArgumentParser(argparse.ArgumentParser):
+    def add_argument(self, *args, **kwargs):
+        action = kwargs.get('action')
+        help = kwargs.get('help')
+        if help and action in {'append', 'append_const', 'count', 'extend', PrependListAction}:
```

Comment:
I've confirmed that `extend` _can_ be provided multiple times.

From the docs (https://docs.python.org/3/library/argparse.html#action):

```
parser = argparse.ArgumentParser()
>>> parser.add_argument("--foo", action="extend", nargs="+", type=str)
>>> parser.parse_args(["--foo", "f1", "--foo", "f2", "f3", "f4"])
Namespace(foo=['f1', 'f2', 'f3', 'f4'])
```